sdk/webpubsub/azure-messaging-webpubsubclient/azure/messaging/webpubsubclient/_client.py

```diff
@@ -353,7 +375,7 @@ def send_to_group(
         :param content: The data content. Required.
         :type content: Any.
```

Comment:
Can we please add overloads to all of the `send` methods so that we don't have to type `content` as Any?
It looks like we have a concrete mapping:
```
content: str, data_type: Literal[WebPubSubDataType.TEXT]
content: JSON, data_type: Literal[WebPubSubDataType.JSON]
content: memoryview, data_type: Literal[WebPubSubDataType.BINARY, WebPubSubDataType.PROTOBUF]
```

Reply:
Make sense